tests/runtests.sh

```diff
@@ -50,6 +69,25 @@ learn-ocaml --sync-dir=/sync --repo=/repository build serve")
 	docker rm "$SERVERID" > /dev/null
 	exit 1
     fi
+
+    if use_client_image; then
+        # Run the client in background
+        CLIENTID=$(docker run --entrypoint '' -d -it --name=client \
+          --add-host host.docker.internal:host-gateway \
+          -v "$(pwd)"/"$dir":/home/learn-ocaml/actual \
+          learn-ocaml-client /bin/sh)
+
+        # Wait for the client to be initialized
+        sleep 2
```

Comment:
That's fragile. We should have a service to determine if the server is ready.

Reply:
Indeed! So I guess I could use the standard docker-compose trick based on `wait-for-it` for the server, but I dunno for the learn-ocaml-client (this second sleep).

Reply:
On second thought it seems I can just get rid of the second sleep for the client (as it's just waiting for `/bin/sh` to be run, which is immediate). I will push another commit soon.